lib/presentation/components/field_picker_types.dart

```diff
@@ -0,0 +1,5 @@
+enum FieldPickerTypes {
```

Comment:
I prefer FieldPickerType, as singular once it's a type.

Reply:
Make sense.